app/src/test/java/org/fossasia/openevent/general/RegExTest.kt

```diff
@@ -0,0 +1,18 @@
+package org.fossasia.openevent.general
+
+import org.fossasia.openevent.general.auth.EditProfileFragment
+import org.junit.Assert.assertEquals
+import org.junit.Test
+
+class RegExTest {
+    val frag = EditProfileFragment()
+    @Test
+    fun `protocol removal when loaded`() {
+        assertEquals("abcdefg", frag.loadStringRegEx("https://instagram.com/abcdefg"))
+    }
+
+    @Test
+    fun `anomaly removal when updating`() {
+        assertEquals("helloworld", frag.updateStringRegEx("h@ello@wo ,   rld"))
```

Comment:
it's invalid!

Reply:
why?

Reply:
it tests the function's use of removing the anomalies... I don't get the problem

Reply:
I can't use that function to make entire links because .. it is also being used by change listener to analyze whether the text is right or not...